pkg/registry/tls_test.go

```diff
@@ -0,0 +1,35 @@
+package registry_test
+
+import (
+	"testing"
+
+	"github.com/google/go-containerregistry/pkg/name"
+	"github.com/google/go-containerregistry/pkg/registry"
+	"github.com/google/go-containerregistry/pkg/v1/random"
+	"github.com/google/go-containerregistry/pkg/v1/remote"
+)
+
+func TestTLS(t *testing.T) {
+	s, err := registry.TLS("test.com")
```

Comment:
I'd usually say to use "example.com" for this, but I feel like that doesn't test that we can pick the domain (since httptest uses example.com itself). Does the hard-coded httptest cert work for "registry.example.com" as well? If not, we could use "registry.example.com" to both avoid using "test.com" and assert that it works for arbitrary domains.

Reply:
Done.